File: app/src/main/java/com/yu/functionbox/utils/LogUtil.java
package com.yu.functionbox.utils;

import android.util.Log;

/**
 * Created by yuw on 2018-01-18.
 * description
 */

public class LogUtil {
    public static void i(Object object,String msg){
        Log.i("yuwei"+object.getClass().getName(),msg);
    }

    public static void e(Object object,String msg){
        Log.e("yuwei"+object.getClass().getName(),msg);
    }
}

File: app/src/main/java/com/yu/functionbox/utils/LogUtil.kt
package com.yu.functionbox.utils

import android.util.Log

/**
 * Created by yuw on 2018-01-18.
 * description
 */

object LogUtil {
    fun i(`object`: Any, msg: String) {
        Log.i("yuwei" + `object`.javaClass.name, msg)
    }

    fun e(`object`: Any, msg: String) {
        Log.e("yuwei" + `object`.javaClass.name, msg)
    }
}
